packages/integrations/useJwt/index.ts

```diff
@@ -0,0 +1,28 @@
+import jwt_decode, { JwtDecodeOptions, JwtHeader, JwtPayload } from 'jwt-decode'
+import { computed, Ref } from 'vue-demi'
+
+/**
+ * Reactive decoced jwt token.
+ *
+ * @see {@link https://vueuse.js.org/integrations/useJwt/}
+ * @param jwt
+ */
+export function useJwt<Payload extends object = JwtPayload, Header extends object = JwtHeader>(encodedJwt: Ref<string>) {
+  const decodeWithFallback = <T extends object>(encodedJwt: string, options?: JwtDecodeOptions) => {
+    try {
+      return jwt_decode<T>(encodedJwt, options)
+    }
+    catch (err) {
+      return Object.assign({})
```

Comment:
```suggestion
      return {}
```

Any reason for this?

Reply:
I am facing type issues when returning a simple `{}`.

<img width="826" alt="image" src="https://user-images.githubusercontent.com/48953604/105904746-cd54b500-6021-11eb-964a-07fd45bc4435.png">

Reply:
I am facing type issues when returning `{}`. E.g. 
```javascript
// Property 'alg' does not exist on type '{}'.
header.value.alg
```

Reply:
Could just use `as` in TypeScript which is compile time. `Object.assign` is runtime.